#497 Allow metrics on custom port with custom host configuration

dnperfors opened the pull request:
This PR allows for serving the metrics to a custom port, but removes the automatic configuration of the host as described in #497 

### Confirm the following

- [X] I have ensured that I have merged the latest changes from the dev branch
- [X] I have successfully run a [local build](https://github.com/AppMetrics/AppMetrics#how-to-build)
- [ ] I have included unit tests for the issue/feature
- [X] I have included the github issue number in my commits

alhardy commented:
[Ticket ](https://github.com/AppMetrics/AppMetrics/issues/628) to remove left over configuration and logging

alhardy reviewed (changes requested):
Could this change please be merged to the AppMetrics:features/4.2.0 branch.

Sorry this isn't clear in the documentation

dnperfors commented:
> Could this change please be merged to the AppMetrics:features/4.2.0 branch.
> 
> Sorry this isn't clear in the documentation

I rebased it and changed the branch to features/4.2.0, initially I did do this change based on that branch, but when I saw the documentation, I rebased it to dev...
